lib/shiba/explain.rb

```diff
@@ -48,13 +48,20 @@ def as_json
         table: @query.from_table,
         md5: @query.md5,
         messages: @result.messages,
+        global: global,
```

Comment:
'global' is kind of generic, not sure what it means. maybe database_name?

Reply:
it's a hash of state that'll be common to all messages.  will eventually include table sizes and other stuff that makes our JSON payload smaller.